src/ansys/geometry/core/tools/prepare_tools.py

```diff
@@ -295,3 +297,93 @@ def enhanced_share_topology(
             share_topo_response.repaired,
         )
         return message
+
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def find_logos(
+        self, bodies: list["Body"] = None, min_height: Real = None, max_height: Real = None
+    ) -> "LogoProblemArea":
+        """Detect logos in geometry.
+
+        Detects logos, using a list of bodies if provided.
+        The logos are returned as a list of faces.
+
+        Parameters
+        ----------
+        bodies : list[Body], optional
+            List of bodies where logos should be detected
+        min_height : real, optional
+            The minimum height when searching for logos
+        max_height: real, optional
+            The minimum height when searching for logos
+
+        Returns
+        -------
+        Logo problem area
+
+            Problem area with logo faces.
+        """
+        from ansys.geometry.core.designer.body import Body
+
+        # Verify inputs
+        if bodies:
+            if len(bodies) > 0:
+                check_type_all_elements_in_iterable(bodies, Body)
+
+        body_ids = [] if bodies is None else [body._grpc_id for body in bodies]
+        find_logo_options = FindLogoOptions()
+        if min_height:
+            find_logo_options.min_height = min_height
+        if max_height:
+            find_logo_options.max_height = max_height
```

Comment:
Couple of things:

1. We are not verifying that the inputs are actually numbers. We should do that.
2. I believe we could simplify this to the following:
```suggestion
        find_logo_options = FindLogoOptions(
            min_height=min_height,
            max_height=max_height,
        )
```

If the values are None, gRPC shouldn't convert them into the message anyway.